packages/main/src/Tokenizer.js

```diff
@@ -215,16 +215,36 @@ class Tokenizer extends UI5Element {
 		this._scrollEnablement.scrollContainer = this.expanded ? this.contentDom : this;
 	}
 
-	_tokenDelete(event) {
+	_delete(event) {
+		if (this._selectedTokens.length) {
+			this._selectedTokens.forEach(token => this._tokenDelete(event, token));
+		} else {
+			this._tokenDelete(event);
+		}
+	}
+
+	_tokenDelete(event, token) {
 		let nextTokenIndex; // The index of the next token that needs to be focused next due to the deletion
-		const deletedTokenIndex = this._getVisibleTokens().indexOf(event.target); // The index of the token that just got deleted
+
+		const tokens = this._getVisibleTokens();
+		const deletedTokenIndex = token ? tokens.indexOf(token) : tokens.indexOf(event.target); // The index of the token that just got deleted
+		const notSelectedTokens = tokens.filter(t => !t.selected);
```

Comment:
I would name it "token" instead of "t".

Reply:
there is already a 'token' variable in the outer scope